app/services/affiliate-slots.js

```diff
@@ -226,7 +227,7 @@ export default Service.extend({
     currentTargeting.forEach((target) => {
       // we're checking all units
       availableUnits.forEach((unit) => {
-        if (unit.campaign === target.campaign && unit.category === target.category) {
+        if (unit.campaign === 'ddb' && unit.category === 'ddb') {
```

Comment:
Can you explain why DnDBeyond has a special case here?

Reply:
It's not, debug code. Thanks for catching